usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -0,0 +1,141 @@
+DROP VIEW IF EXISTS award_delta_view;
+
+CREATE VIEW award_delta_view AS
+WITH views AS (
```

Comment:
Did you look at using `vw_award_search` instead of re-creating unions of the matviews?

If this CTE is necessary, I recommend using `UNION ALL` instead of `UNION` for performance reasons. There will not be duplicate awards in the various matviews.